erica_app/erica/request_processing/erica_input.py

```diff
@@ -6,7 +6,9 @@
 
 
 class FormDataEst(BaseModel):
-    steuernummer: str
+    steuernummer: Optional[str]
+    new_admission: Optional[bool]
```

Comment:
Yes, I was also thinking about sticking to the German name "Neuaufnahme". Personally, I would go with `submission_without_tax_nr ` because `request_tax_nr ` sounds as if we will send a tax_nr back from Erica

Reply:
Hm, a tricky line to toe there: "Neuaufnahme" is probably what the documentation says, but I _also_ wouldn't really know at a glance that this means "this person has no tax nr". (It's different for technical terms like "bufa_nr" that are unambiguously in the tax domain.)

In this case, I'd go for a name that is clear to us, even if it doesn't match the German documentation.

Reply:
Let's use `submission_without_tax_nr`